DP02_09_Custom_Coadds/DP02_09b_Custom_Coadd_Sources.ipynb

```diff
@@ -345,371 +336,329 @@
+   "id": "c0b5829c-bbad-4d31-a8e5-aeee1faa69d8",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "No special configurations are used for the source measurements."
+    "yaml_file = '$DRP_PIPE_DIR/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml'\n",
+    "steps = 'detection,mergeDetections,deblend,measure'\n",
+    "my_uri = yaml_file + '#' + steps\n",
+    "print(my_uri)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "c1d7f1b4-6af9-41a4-b209-5b0ec66c53ac",
+   "cell_type": "markdown",
+   "id": "6164a9c0-2d71-4539-9228-c0dc3bdb5004",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "sourceMeasurementTask = SingleFrameMeasurementTask(schema=schema,\n",
-    "                                                   algMetadata=algMetadata)"
+    "Note how the URI begins with the full DP0.2 DRP YAML pipeline definition file name, then, after a `#` character, continues with a comma-separated list of the desired pipeline steps. The order in which these steps are listed after the `#` character does not matter, though here they are listed in the order they run during processing for the sake of understandability."
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "ee27ace8-c5f5-464a-b357-8661b58df763",
+   "id": "ecc7fb86-c887-44ec-a7e7-54f802ca0473",
    "metadata": {},
    "source": [
-    "**Option:** the help documentation can be accessed with the `help()` command."
+    "Now create a source detection pipeline object based on the URI you just constructed:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "e4492fa2-fed5-4124-8815-274e081c68b8",
+   "id": "6e84ab85-3420-4f74-847c-d8e66dc36559",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# help(sourceMeasurementTask.ConfigClass())"
+    "sourceDetectionPipeline = Pipeline.from_uri(my_uri)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "3c79e4c6-f5ad-425b-92b0-f791b51fb6b4",
+   "id": "5397e4f5-db25-4f85-8730-d18d062c1d14",
    "metadata": {},
    "source": [
-    "## 2.3. Characterize image\n",
-    "\n",
-    "Send the standard output (a structure) into `result_ci` to avoid having it display to screen (see Section 2.4 for more about structures).\n",
-    "\n",
-    "The following cell will still provide standard output information about progress of the task."
+    "As in DP0.2 tutorial notebook 09a, it's necessary to apply a small number of configuration overrides to customize the pipeline's operation. The first two configuration overrides below set the detection threshold applied to your custom i-band coadd to be 10 sigma. The `multibandDeblend.maxIter` configuration override sets the maximum deblending iterations to a lower value than its default, which is a recommended mechanism for speeding up the processing. The `doPropagateFlags` configuration override is set because this processing does not need to propagate flag information about which sources were used for PSF construction."
```

Comment:
Re. the configuration overrides, we should say something about why 10 sigma.... I think it's just to provide an example of doing something different from the default? That's a fine reason but we should say that, so that the use of 10 is not interpreted as a standard or a recommendation.